backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -218,6 +218,16 @@ def admin_logout_link
         end
       end
 
+      def stores_formatted_for_select
```

Comment:
```suggestion
      def stores_dropdown_values
```

Reply:
Going to submit this as a separate commit to test the random failing test.